handlers/secrets.go

```diff
@@ -65,7 +111,8 @@ func getSecretsHandler(kube typedV1.SecretInterface, w http.ResponseWriter, r *h
 	secrets := []types.Secret{}
 	for _, item := range res.Items {
 		secret := types.Secret{
-			Name: item.Name,
+			Name:      item.Name,
+			Namespace: item.Namespace,
```

Comment:
This line is incorrect and is why the tests are currently failing, the `faas-provider` needs to be updated, the latest version is `0.12.0`. This needs to be updated in the `Gopkg.toml` and then you need to run `dep ensure` again 


```go
// Secret for underlying orchestrator
type Secret struct {
	Name  string `json:"name"`
	Value string `json:"value,omitempty"`
}
```

Reply:
@jonatasbaldin PTAL

Reply:
Done.